README.md

```diff
@@ -204,6 +204,13 @@ duplicate heads.
 Alternatively, you can use the [head2branch plugin](./plugins/head2branch)
 to create a new named branch from an unnamed head.
 
+In some projects, heads are closed without being merged (with a close
+branch commit), whether as a way to re-write history or just simply
+abandoned changes. In these cases, it may be appropriate to simply ignore
```

Comment:
The first sentence in this paragraph is strange. Why not drop the description of when Mercurial creates multiple heads and just add a sentence to the previous paragraph:

"hg-fast-export can be made to ignore unnamed heads with the `-ignore-unnamed-heads` option.

Reply:
My goal was to make it clear the scenario where it is appropriate to use the option. Another case for multiple heads is when you use bookmarks. I don't know if hg-fast-export sees those as branches or not. But, in the case of bookmarks you most definitely do not want to use `-ignore-unnamed-heads`. In the repos I was converting, it was appropriate because all of the extra heads were close commits (which the script does not seem to treat specially).

I agree with combining with earlier paragraph. In my latest commit I combined and re-wrote it to be more like what you proposed.

Ultimately, though, I'm just trying to improve the docs, so I can change it to whatever you want for the PR to complete.

Thanks for working on this script, it is a lot better than the Mercurial plugin and I'm using it to convert multiple repos to git.

Reply:
The new version is much better. I didn't get that the point you were trying to make was that `-ignore-unnamed-heads` was only appropriate for abandoned and unmerged changes and only then. Breaking it up into a numbered list makes it much clearer!